src/routes/(app)/admin[[hash=admin_hash]]/product/alias/+page.svelte

```diff
@@ -1,11 +1,32 @@
 <script lang="ts">
+	import { enhance } from '$app/forms';
+
 	export let data;
+	let listAliases = data.products.flatMap((product) => product.alias?.[1]);
+
+	let errorDuplicateAlias = false;
+	let loading = false;
 </script>
 
 <h1 class="text-3xl">Bulk Alias Change</h1>
 
-<form class="flex flex-col gap-2" method="post">
-	{#each data.products as product}
+<form
+	class="flex flex-col gap-2"
+	method="post"
+	use:enhance={() => {
+		errorDuplicateAlias = false;
+		return async ({ result }) => {
+			loading = false;
+
+			if (result.type === 'error') {
+				errorDuplicateAlias = true;
```

Comment:
I think we should just display the error message from the backend (and make sure the backend shows a readable error message)